File: lib/NCU.AnnualWorks.Authentication.JWT.Core/Models/UserClaims.cs
namespace NCU.AnnualWorks.Authentication.JWT.Core.Models
{
    public class UserClaims
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string AvatarUrl { get; set; }
        public bool IsParticipant { get; set; }
        public bool IsLecturer { get; set; }
        public bool IsAdmin { get; set; }
        public bool IsCustom { get; set; }
    }
}

File: src/NCU.AnnualWorks/ClientApp/src/shared/models/Auth/UserClaims.ts
export type UserClaims = {
  Id: string;
  Name: string;
  Email: string;
  IsParticipant: string;
  IsLecturer: string;
  IsAdmin: string;
  IsCustom: string;
  AvatarUrl: string;
  iss: string;
  exp: number;
  nbf: number;
  iat: number;
};

export default UserClaims;